### PR TITLE
feat: add actions to publish npm automatically

### PR DESCRIPTION
**Pull request by zkd8907**
**Checklist:**

- [x] test cases has added or updated
- [x] documentation has added or updated
- [x] commit message follows the [convention commit guidelines](https://conventionalcommits.org/)

- 使用Github Actions替换travis
- 使用Github Actions完成自动发布npm

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/TSW?pullRequest=347) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/TSW?pullRequest=347) before we can accept your contribution.<br/><hr/>**lemanzhang** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/TSW?pullRequest=347) it.</sub>

**Comment by zkd8907** [comment]
> `0.0.0-development` 这个验证了是可以不改 package.json 里面 version 来发版本的是吗？

测试过。

**Comment by zkd8907** [comment]
> `0.0.0-development` 这个验证了是可以不改 package.json 里面 version 来发版本的是吗？

image

验证Ok，semantic-release会处理version的差异，并且以计算出来的version打[tag](https://github.com/zkd8907/TSW/tags)以及发布[npm包](https://www.npmjs.com/package/tsw-zkd8907)

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@tswjs/tsw",
-  "version": "1.4.3",
+  "name": "tsw-zkd8907",
```
Comment (mapleeit) [inline package.json]: 这里和下面 `repository.url` 记得改回来

Comment (zkd8907) [reply package.json]: Already fixed

**Pull request merged**